de-lib/src/main/java/org/iplantc/de/theme/base/client/apps/cells/AppRatingCellDefaultAppearance.java

```diff
@@ -181,6 +188,13 @@ public void render(SafeHtmlBuilder sb, App app) {
         int total = app.getRating().getTotal();
         // Build five rating stars
         for (int i = 0; i < ratings.size(); i++) {
+            if (!app.isPublic() || app.getAppType().equalsIgnoreCase(App.EXTERNAL_APP)) {
+                sb.append(templates.imgCell("Rating-" + i,
+                                            ratings.get(i),
+                                            resources.css().disabledRating(),
+                                            resources.whiteStar().getSafeUri()));
```

Comment:
Should the Star even be shown? Looking at these changes, it almost seems like a bug that hovering over the stars doesn't work.